packages/itwin/breakdown-trees/package.json

```diff
@@ -77,7 +77,7 @@
     "chai-as-promised": "^7.1.1",
     "chai-jest-snapshot": "^2.0.0",
     "chai-spies": "1.0.0",
-    "cpx": "^1.5.0",
+    "cpx2": "^3.0.0",
```

Comment:
Why do you need to make this change in this package?

Reply:
Security vulnerability. 
I should've caught this last time.